app/src/main/res/values/strings.xml

```diff
@@ -311,7 +311,7 @@
 
   <string name="inappnotif_set_up_pin">Your wallet is not protected.</string>
   <string name="inappnotif_set_up_pin_action">Set a PIN</string>
-  <string name="inappnotif_mnemonics_never_seen">Thanks for using Phoenix. Have you made a backup of your wallet?</string>
+  <string name="inappnotif_mnemonics_never_seen">Thanks for using Phoenix. Please backup your wallet so that you don\'t lose your bitcoins.</string>
```

Comment:
The translation to Portuguese Brazilian: Obrigado por usar a Phoenix. Faça o backup de sua carteira para não correr risco de perder seus bitcoins.

Reply:
fixed with 53d9c35